<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>GroupBox</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>2</number>
        </property>
        <property name="leftMargin" >
         <number>4</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="bottomMargin" >
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>navn:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>adresse1:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>adresse2:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>post &amp; by</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>land</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="addResponce" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Tilføj Svar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="RKTreeWidget" name="treeWidget" >
       <column>
        <property name="text" >
         <string>RMA nr.</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QCheckBox" name="checkBox" >
         <property name="text" >
          <string>Har Parent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2" >
         <property name="text" >
          <string>Har Child</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_3" >
         <property name="text" >
          <string>curNode Changed</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string> RMA Data </string>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="onrLabel" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dateLabel" >
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="eMailLabel" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="stkLabel" />
            </item>
            <item>
             <widget class="QLabel" name="vnrLabel" />
            </item>
            <item>
             <widget class="QLabel" name="vtxtLabel" />
            </item>
            <item>
             <widget class="QLabel" name="prisLabel" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="estkLabel" />
            </item>
            <item>
             <widget class="QLabel" name="evnrLabel" />
            </item>
            <item>
             <widget class="QLabel" name="evtxtLabel" />
            </item>
            <item>
             <widget class="QLabel" name="eprisLabel" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="title" >
        <string> Tilføj/Send Data </string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="ddateLabel" >
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="dataMsg" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="sendRMA" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RKTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>RKTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
